src/capybara.js

```diff
@@ -269,8 +269,20 @@ Capybara = {
     return true;
   },
 
+  sendKeys: function (index, keys) {
+    var strindex, length;
+
+    length = keys.length;
+
+    this.focus(index);
```

Comment:
Yeah, I just put it back. I guess a little duplication isn't the end of the world.